lib/features/support/option.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Provides the set of Maze Runner command line options
+class Option
+  # Common options
+  SEPARATE_SESSIONS ||= 'separate-sessions'
+  FARM ||= 'farm'
+  APP ||= 'app'
+  A11Y_LOCATOR ||= 'a11y-locator'
+  RESILIENT ||= 'resilient'
+
+  # BrowserStack-only options
+  BS_LOCAL ||= 'bs-local'
+  BS_DEVICE ||= 'device'
+  USERNAME ||= 'username'
+  ACCESS_KEY ||= 'access-key'
+  BS_APPIUM_VERSION ||= 'appium-version'
+
+  # Local-only options
+  OS ||= 'os'
+  OS_VERSION ||= 'os-version'
+  APPIUM_SERVER ||= 'appium-server'
+  APPLE_TEAM_ID ||= 'apple-team-id'
+  UDID ||= 'udid'
```

Comment:
I don't think these need to be `||=`

```suggestion
  # Common options
  SEPARATE_SESSIONS = 'separate-sessions'
  FARM = 'farm'
  APP = 'app'
  A11Y_LOCATOR = 'a11y-locator'
  RESILIENT = 'resilient'

  # BrowserStack-only options
  BS_LOCAL = 'bs-local'
  BS_DEVICE = 'device'
  USERNAME = 'username'
  ACCESS_KEY = 'access-key'
  BS_APPIUM_VERSION = 'appium-version'

  # Local-only options
  OS = 'os'
  OS_VERSION = 'os-version'
  APPIUM_SERVER = 'appium-server'
  APPLE_TEAM_ID = 'apple-team-id'
  UDID = 'udid'
```

Reply:
Without them you get loads of `warning: already initialized constant` messages comes out, because of the way that Ruby files are loaded into Cucumber (using the `-r`option). I started to look at ways of doing that differently, but it didn't seem to gain traction.

Reply:
Is that because cucumber is loading all the files in `lib/features/...`? So would putting this in `lib/maze/option.rb` fix it?

I think it'd be better to avoid having cucumber load files that it doesn't need to

Reply:
Yes, that makes sense.  I'll do this on the next branch as well, as I have some further moves coming and it would be good to avoid awkward rebases for now.